tests/fixtures/sample_lib/modules/file_processing/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Local
```

Comment:
NIT: Copyright header here too (and anywhere else that's new)! We really need to add a CI check for this

Reply:
Added! Do only `.py` files need it or should I also add a copyright header  to [config.yaml](https://github.com/caikit/caikit/blob/bb73b8a0bd260d18d7dff7d2f154ab8a81f049c5/tests/fixtures/models/box/config.yml)?

Reply:
@HonakerM You will need to add the copyright to all files, including `yml` files!